docs/api.md

```
@@ -6,11 +6,13 @@ WebUI が使用する REST API および WebSocket の仕様。
 1. [認証](#1-認証)
 2. [REST API](#2-rest-api)
-   - [ヘルスチェック](#21-ヘルスチェック)
-   - [設定](#22-設定)
-   - [セッション](#23-セッション)
-   - [メッセージ履歴](#24-メッセージ履歴)
-   - [ストリーミングチャット](#25-ストリーミングチャット)
+    - [ヘルスチェック](#21-ヘルスチェック)
+    - [Readiness](#22-readiness)
+    - [メトリクス](#23-メトリクス)
+    - [設定](#24-設定)
+    - [セッション](#25-セッション)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)